lib/niconico/video.rb

```diff
@@ -101,6 +105,24 @@ def get_video
       end
     end
 
+    def get_comment(num, owner=false)
+      raise CommentUnavailableError unless @ms
+
+      params = {}
+      params[:version] = 20090904
+      params[:res_from] = -1 * num
+      params[:thread] = @getflv_thread_id
+      params[:fork] = 1 if owner
+
+      begin
+        res = @agent.get(File.join(@ms, "thread"), params)
```

Comment:
Don't use `File.join` for concatenating URL paths. It's for file path and depending on `File::SEPARATOR`. Not for URL.